pom.xml

```diff
@@ -353,13 +353,13 @@ under the License.
 			<dependency>
 				<groupId>org.apache.flink</groupId>
 				<artifactId>flink-shaded-netty</artifactId>
-				<version>4.1.82.Final-${flink.shaded.version}</version>
+				<version>4.1.91.Final-${flink.shaded.version}</version>
```

Comment:
We initially thought of doing the flink-shaded update early in the release to have enough CI cycles to test the functionality. With the delays we're getting close to the 1.18 feature freeze which contradicts the initial idea of the timing. I discussed this with @zentol: Based on past experience, netty was the source of instabilities. So, if we want to play it safe we could upgrade all the shaded dependencies except for netty and do the netty upgrade in 1.19 early on.

But on the other note: With 2 weeks until the feature freeze (which might be postponed anyway) and additional time for bug fixing and release testing, we still might have enough CI cycles. What's your opinion on that one?

I don't see a reason to not do the upgrade more conservatively (i.e. waiting with the netty upgrade and doing it in a separate ticket). Chesnay things that there is still enough time.

Reply:
> With 2 weeks until the feature freeze (which might be postponed anyway) and additional time for bug fixing and release testing, we still might have enough CI cycles. What's your opinion on that one?

I think there's still enough time to actually merge the changes as-is.

Reply:
Ok, fine with me - I just wanted to bring this discussion up once. :+1: